packages/@guidesmiths/cuckoojs-cli/src/lib/runners/schematic.runner.ts

```diff
@@ -12,38 +12,33 @@ export class SchematicRunner extends GenericRunner {
 		super('node');
 	}
 
-	public async generateNestApplication(name: string) {
-		const args = ['@nestjs/schematics:application', `--name ${name} --verbosity=QUIET -`];
-		await super.run({command: SchematicRunner.getSchematicsCliPath(), args, stdio: ['pipe', 'pipe', 'inherit']});
-	}
-
 	public async generateNestElement(schematic: string, name: string, options: string[]) {
 		const args = [`@nestjs/schematics:${schematic} ${name} ${options.join(' ')}`];
-		await super.run({command: SchematicRunner.getSchematicsCliPath(), args, stdio: ['pipe', 'pipe', 'inherit']});
+		await super.run({command: SchematicRunner.getSchematicsCliPath(), args});
 	}
```

Comment:
As we're now using the nest cli, perhaps we can also replace this by calling to the corresponding cli call in a future refactor.

Reply:
My intention is to leave this class only for schematics (mainly `cuckoojs-schematics`. This function will be replaced with one in the `NestRunner`, which will use the `nestjs-cli`. I thought of leaving it for the next PR as it would imply to also modify the `nest generate` command

Reply:
I agree :)